assets/less/icons.less

```diff
@@ -9,6 +9,17 @@
   font-style: normal;
 }
 
+@font-face {
+  font-family: 'remixicon';
```

Comment:
We could use the `remixicon` npm package and import the corresponding .css file, less files to maintain :)

Sidenote, we currently load fonts form CDN, but I guess for offline usage it's ok if the font doesn't load, while we still want the icons to be there.

Reply:
Would npm generate files with only the icons we use?

Reply:
Nah, it's just that we wouldn't need to embed the files manually.

Reply:
The remix is quite large. I think that for our case where we use a few icons, doing it manually fits us well.

Reply:
Oh you're including only the subset, ignore me then!